normative/dtpr-stickers
# Supports logo as img, ref or placeholder svg from local files

# Notion card link(s)
- [Convert image refs to URLs](https://www.notion.so/normative/Convert-image-refs-to-URLs-b38f2d16c4e74ae5b7a4fb6943ed2133)

# Review APP
- [Heroku Review App](https://dtpr-sticker-hotfix-pri-i5ix1y.herokuapp.com/sensors/slcJ3Jk93uilfXcZaOqhG)

# PR Description (Proposed changes, approach, gotchas, etc)
- [x] Handles not found sensor for print page
- [x] Allows accountable logo comes from ref, link and placeholder
- [x] Removes sensor caching

# How to test it
- Access Review App through the link on the top of description then:
  - try a print page with wrong sensor id
  - try multiple sensor with different sensors `logoRef` and see print page and also sensor page
  - change sensor information like some tech type or something else then refresh sensor page
